app/views/wizards/claims/add_mentor_wizard/_mentor_step.html.erb

```diff
@@ -0,0 +1,38 @@
+<% caption = current_user.support_user? ? "#{t(".caption")} #{contextual_text}" : t(".caption") %>
+<% content_for :page_title, current_step.errors.any? ? t(".page_title_with_error", error: current_step.errors.first.message, contextual_text: caption) : t(".page_title", contextual_text: caption) %>
```

Comment:
We can simplify this with https://govuk-components.netlify.app/helpers/title-with-error-prefix/. Check what I done on https://github.com/DFE-Digital/itt-mentor-services/pull/1171/files